### PR TITLE
Adding associatedJudge to the CaseDeadline entity

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: shared/src/business/entities/CaseDeadline.test.js [hunk 1]
```diff
@@ -57,5 +58,59 @@ You ain't been up off that porch, now`,
...
+        {
+          caseId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+          deadlineDate: '2019-03-01T21:42:29.073Z',
+          description: 'One small step',
+        },
+        { applicationContext },
+      );
+
+      expect(caseDeadlineWithJudge.associatedJudge).toEqual(mockJudgeName);
+      expect(caseDeadlineWithoutJudge.associatedJudge).toEqual(
```
Comment (matthopson) [inline shared/src/business/entities/CaseDeadline.test.js]: Should this be here?

File: shared/src/business/entities/CaseDeadline.test.js [hunk 2]
```diff
@@ -57,5 +58,59 @@ You ain't been up off that porch, now`,
...
+      const caseDeadlineWithoutJudge = new CaseDeadline(
+        {
+          caseId: 'c54ba5a9-b37b-479d-9201-067ec6e335bb',
+          deadlineDate: '2019-03-01T21:42:29.073Z',
+          description: 'One small step',
+        },
+        { applicationContext },
+      );
+
+      expect(caseDeadlineWithJudge.associatedJudge).toEqual(mockJudgeName);
```
Comment (matthopson) [inline shared/src/business/entities/CaseDeadline.test.js]: Also this. It looks like these two tests are doing the same thing.

File: shared/src/business/entities/CaseDeadline.test.js (hunk 1 above)
Comment (rachelschneiderman) [reply shared/src/business/entities/CaseDeadline.test.js]: Good catch!

**Review by matthopson — APPROVED** [review]
ship-it-brown

**Review by ericsorenson — APPROVED** [review]

**Review by kkoskelin — APPROVED** [review]

**Review by wbyoko — APPROVED** [review]

**Pull request merged**